standard_reports/README.md

```diff
@@ -247,7 +252,58 @@ Other functionality, and processes used in report development are:
 
 ### Tera templating language
 
-Open mSupply reports are rendered using [Tera](https://keats.github.io/tera/docs/), an extension of HTML where values are replaced during render and simple logic can be executed.
+OMS forms and reports are rendered as HTML using [Tera](https://keats.github.io/tera/docs/), an extension of HTML where values are replaced during render and simple logic can be executed.
+
+### Excel
+
+Reports & forms are also exportable to Excel format. This is done by parsing the HTML template and converting it to an Excel worksheet.
+
+#### Parsable HTML templates
+
+By default, only the main body of the report (i.e. `template.html`) is parsed to create the Excel worksheet. To be converted to Excel, this template must use HTML <table> elements.
+
+- The `template.html` main body template should have a `<table>`
+  - Then for row headers > `<thead>` > `<tr>` > `<th>`
+  - And for data rows > `<tbody>` > `<tr>` > `<td>`
+
+#### Customising the Excel export
+
+##### Header section
+
+The `header.html` template often includes important info (report title, date, customer etc.). Map these to the Excel sheet by using the `excel-cell` attribute.
+
+`excel-type="title"` and `excel-type="bold"` are available for simple styling.
+
+```html
+<div>
+  <h1 excel-cell="A1" excel-type="title">Report Title</h1>
+  <div>
+    <span excel-cell="C2" excel-type="bold">Date:</span>
+    <span excel-cell="D2">{{ current_date }}</span>
+  </div>
+  <div>Unrelated for Excel</div>
+</div>
+```
+
+would map to:
+
+|     | A                                                     | B   | C         | D      |
+| --- | ----------------------------------------------------- | --- | --------- | ------ |
+| 1   | <span style="font-size: 18px">**Report Title**</span> |     |           |
+| 2   |                                                       |     | **Date**: | 1/1/26 |
+
+Main data rows would be rendered starting 1 row after the lowest used row - in this case we'd start from row 4.
```

Comment:
I can't quite understand this bit, can you please rephrase or explain with example ?

Reply:
I would imagine we would need excel-row attribute at some point to specify where table needs to start (for templates ?)